eng/Versions.props

```diff
@@ -334,8 +334,8 @@
     <XamarinMacOSWorkloadManifestVersion>15.0.9600-net9-rc2</XamarinMacOSWorkloadManifestVersion>
     <XamarinTvOSWorkloadManifestVersion>18.0.9600-net9-rc2</XamarinTvOSWorkloadManifestVersion>
     <!-- Workloads from dotnet/emsdk -->
-    <MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportPackageVersion>9.0.0-rc.2.24468.8</MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportPackageVersion>
-    <EmscriptenWorkloadManifestVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportPackageVersion)</EmscriptenWorkloadManifestVersion>
+    <MicrosoftNETWorkloadEmscriptenCurrentManifest90100PackageVersion>9.0.0</MicrosoftNETWorkloadEmscriptenCurrentManifest90100PackageVersion>
```

Comment:
We still need this in 9.0.200 because that uses the 9.0.100 manifests, right?

Reply:
Not quite sure I understand the question. We used the TransportPackage before because it was a stable version of an unstable package (ie every preview the package id for the emsdk package would change so that would require manual intervention each time. Larry solved this by wrapping it in a transport package that had a stable ID.) Now that we're in GA for workloads for 9, the package ID isn't' changing anymore so no need to use the transport package anymore.

Make sense?